thinkhazard/static/less/common.less

```diff
@@ -37,6 +37,13 @@ ul.horizontal li {
 .navbar {
   border: 0;
   .box-shadow(0px 2px 10px -1px rgba(0, 0, 0, 0.33));
+  li:last-child a{
+      padding-top: 0px;
+      padding-bottom: 0px;
+      margin-top: 29px;
+      margin-bottom: 29px;
+      border-left: 1px solid #d9d9d9;
```

Comment:
Do you mean adding the separator by adding a divider-vertical class instead of accessing the last element? This is what is don in the link you provided, but in the end they also add a border In the links you they also add it as a border.